configs/openSUSE/dbus-services.toml

```diff
@@ -1112,3 +1112,13 @@ nodigests = [
     "/usr/share/dbus-1/system-services/org.fedoraproject.SetroubleshootFixit.service",
     "/usr/share/dbus-1/system-services/org.fedoraproject.SetroubleshootPrivileged.service"
 ]
+
+[[FileDigestGroup]]
+package = "kcron"
+type = "dbus"
+note = "a priviledge separation helper to modify system crontab"
```

Comment:
Apart from this typo in `priviledge` all looks good :-)

Reply:
Perfect, let me fix this